scripts/autotracking.lua

```diff
@@ -0,0 +1,416 @@
+-- Configuration --------------------------------------
```

Comment:
Welcome to my personal hell mapping every single hex location of the chests in the game